app/LocalServer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+require_once 'Autoloader.php';
+
+use App\Database\DBConnection;
+use App\Exception\HttpException;
+use App\Logger\Handler\LogHandler;
+use App\Logger\Logger;
+use App\Repositories\WordRepository;
+use App\Routes\RouteManager;
+use App\Services\FileService;
+
+header("Content-Type: application/json; charset=UTF-8");
```

Comment:
Move this into class near response itself maybe? I see no point of adding it here, unless there is a point? 😄